Sources/Control/Volume.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  Volume.swift
+//  ControlKit
+//
+
+import MediaPlayer
+
+public extension Control {
+    
+    /// Control the system volume using the underlying `MPVolumeView`.
+    @MainActor
+    enum Volume {
+        
+        /// Subscribe to `volume` via ``VolumeController/volume`` `@Published` property.
+        ///
+        /// - Important: Updating this property will _unmute_ the system volume. The volume level prior to being muted will
+        /// be ignored when setting the volume via this property.
+        public static var volume: Float {
+            get {
+                MPVolumeView.volume
+            }
+            set {
+                if newValue != 0 && isMuted {
+                    isMuted.toggle()
+                }
```

Comment:
Would you like to maintain the `isMuted` property set to `true` when the `volume` is explicitly set to 0?

Reply:
I'm trying to wrap my head around this one and I think the code matches what you describe? 🤔

The idea is "unmute if not zero" - I've added a comment above the `if`.